src/Take.Blip.Builder/FlowManager.cs

```diff
@@ -148,6 +157,14 @@ public async Task ProcessInputAsync(Message message, Flow flow, CancellationToke
                         var stateId = await _stateManager.GetStateIdAsync(context, linkedCts.Token);
                         state = flow.States.FirstOrDefault(s => s.Id == stateId) ?? flow.States.Single(s => s.Root);
 
+                        // If current stateId of user is different of inputExpiration stop processing
+                        if (!_inputExpirationHandler.ValidateState(state, message))
```

Comment:
I know this name was suggest by myself, but I usually name booleans with 'question prefixes' like Is/Can/Has. So here a better name would be `IsValidState` or even `ShouldStopProcessing` (with this option the logic is reversed on the method and the `!` negation won't be needed).

Reply:
In this option `ShouldStopProcessing` i believe that not represents what the method do.